### PR TITLE
Don't use worldbounds for PVS

### PR DESCRIPTION
**Pull request by metalgearsloth**
@PaulRitter 

Don't look until exams done or I cut you.

Main goal is if a grid is spinning then no new chunks should be going in range as a result.

### CODE REVIEW

File: Robust.Shared.Maths/Vector2.cs [hunk 1]
```diff
@@ -130,6 +130,12 @@ public Vector2(float x, float y)
             return new((int) MathF.Floor(X), (int) MathF.Floor(Y));
         }
 
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
+        public readonly Vector2i Ceiling()
```
Comment (PaulRitter) [inline Robust.Shared.Maths/Vector2.cs]:
```suggestion
        public readonly Vector2i Ceiled()
```

File: Robust.Server/GameStates/ChunkIndicesEnumerator.cs [hunk 2]
```diff
@@ -5,30 +5,31 @@ namespace Robust.Server.GameStates;
...
-        _topLeft = (viewBox.TopLeft / chunkSize).Floored();
-        _bottomRight = (viewBox.BottomRight / chunkSize).Floored();
+        _bottomLeft = ((viewPos - range) / chunkSize).Floored();
+        // Also floor this as we get the whole chunk anyway.
+        _topRight = ((viewPos + range) / chunkSize).Floored();
 
-        _x = _topLeft.X;
-        _y = _bottomRight.Y;
+        _x = _bottomLeft.X;
+        _y = _topRight.Y;
```
Comment (PaulRitter) [inline Robust.Server/GameStates/ChunkIndicesEnumerator.cs]: danger danger. this one should be `bottomLeft`

**Review by PaulRitter — CHANGES_REQUESTED** [review]
get owned nerd

File: Robust.Server/GameStates/ChunkIndicesEnumerator.cs (hunk 2 above)
Comment (metalgearsloth) [reply Robust.Server/GameStates/ChunkIndicesEnumerator.cs]: A

File: Robust.Shared.Maths/Vector2.cs (hunk 1 above)
Comment (metalgearsloth) [reply Robust.Shared.Maths/Vector2.cs]: Ceiled's not even a word m8

**Review by PaulRitter — APPROVED** [review]

**Comment by metalgearsloth** [comment]
This will probably be superseded by whatever gets used for https://github.com/space-wizards/RobustToolbox/issues/2527 but okay for now.

**Pull request merged**